packages/graphback-keycloak-authz/tests/KeycloakCrudService.test.ts

```diff
@@ -205,11 +210,11 @@ test('multiple roles can be applied to each operation', async () => {
 
   const context = { kauth: new KeycloakContextBase(Token) }
 
-  const val = {test: 'value'}
+  const val = { test: 'value' }
```

Comment:
Not related, but I think we should automate this to avoid these linting noises in a PR :-)